src/components/inputs/multi-select/multi-select.component.tsx

```diff
@@ -80,7 +80,7 @@ const MultiSelect: React.FC<FormFieldInputProps> = ({ field, value, errors, warn
     return field.isRequired ? <FieldLabel field={field} /> : <span>{t(field.label)}</span>;
   }, [field.isRequired, field.label, t]);
 
-  return sessionMode == 'view' || sessionMode == 'embedded-view' ? (
+  return sessionMode == 'view' || sessionMode == 'embedded-view' || isTrue(field.readonly) ? (
```

Comment:
Ditto

Reply:
For this component we're using `CheckboxGroup` and it has a finicky behavior where regardless of if that property is passed, the user can still interact with the input field. 

We might have to consider refactoring this component to use `MultiSelect` from Carbon later on that has support for this natively but for now this defaults it to the same display as the view only mode.